launcher-common/src/main/java/org/apache/brooklyn/launcher/common/BasicLauncher.java

```diff
@@ -724,6 +725,15 @@ protected void startApps() {
                 try {
                     LOG.info("Starting brooklyn application {} in location{} {}", new Object[] { app, locations.size()!=1?"s":"", locations });
                     ((Startable)app).start(locations);
+                    Entities.dumpInfo(app);
+                    String sensors = "";
+                    if (app.getAttribute(Attributes.MAIN_URI_MAPPED_PUBLIC)!=null) {
+                        sensors = ": "+app.getAttribute(Attributes.MAIN_URI_MAPPED_PUBLIC);
+                    } else if (app.getAttribute(Attributes.MAIN_URI)!=null) {
+                        sensors += ": "+app.getAttribute(Attributes.MAIN_URI);
+                    }
+                    LOG.info("Started brooklyn application {} in location{} {}{}", new Object[] { app, locations.size()!=1?"s":"", locations,
+                        sensors });
```

Comment:
I thought this is executed in classic mode only, no? How is the app passed to the osgi launcher?